test/hummingbot/connector/exchange/mexc/test_mexc_api_user_stream_data_source.py

```diff
@@ -1,15 +1,15 @@
 import asyncio
 from collections import Awaitable
+from test.hummingbot.connector.network_mocking_assistant import NetworkMockingAssistant
```

Comment:
This import is in the wrong place. We should correctly sorted unless it has been moved due to import issues.